csrc/ops/composite.cpp

```diff
@@ -544,11 +544,24 @@ SdpfaFwdResult sdpfa_fwd(
   TensorView* log_sumexp =
       IrBuilder::create<TensorView>(log_sumexp_td, DataType::Float);
 
+  TensorView* philox_seed = nullptr;
+  TensorView* philox_offset = nullptr;
+#if NVF_TORCH_VERSION_NO_LESS(2, 7, 0)
+  // API changes in torch 2.7.0
+  // The torch API returns philox_seed -> rng_state (uint64_t[2])
+  // and philox_offset -> _unused (empty tensor)
+  philox_seed = TensorViewBuilder()
+                    .shape(std::vector<int64_t>{2})
```

Comment:
nitpick: do we need to be explicit with `std::vector<int64_t>{2}`? I think we usually just do `shape({2})`.

Reply:
Yes. Surprisingly, I was getting an ambiguous construct compile error. I did not dig into it more.